src/features/items/atoms/noItems/NoItemsToDisplay.tsx

```diff
@@ -3,12 +3,13 @@ import { Image } from '@nextui-org/react';
 import { useTranslations } from 'next-intl';
 
 import { Item } from '@/features/items/interfaces/itemsResponse';
+import { User } from '@/features/navbar/organisms/NavbarTop';
 import { icons } from '@/features/shared/hooks/icons';
 
 import style from './no-items-to-display.module.css';
 
 type Props = {
-  data: Item[];
+  data: Item[] | User[];
```

Comment:
Esto no esta bien, imaginate si agregamos 5 vistas mas que usen este componente, tendriamos que venir a el tipo y agregar esto, no escala.

1) Podrias usar genericos para tipar esto.
2) Aca estamos en usuarios, asi que el texto de "No items to display" no seria correcto. Hay que mandar por props el texto a mostrar.